components/segmenter/src/complex/mod.rs

```diff
@@ -41,17 +41,13 @@ pub(crate) struct ComplexPayloads {
 }
 
 #[cfg(feature = "lstm")]
-const MY_LSTM: &DataMarkerAttributes =
-    DataMarkerAttributes::from_str_or_panic("Burmese_codepoints_exclusive_model4_heavy");
+const MY_LSTM: &DataMarkerAttributes = DataMarkerAttributes::from_str_or_panic("Burmese");
```

Comment:
Nit: Prefix matches need to have a separator like `Burmese_`

Real-life example: Malay and Malayalam, which are both major languages (Malaysia and India). There are a dozen more conflicts if you include minor languages.